agentops/client.py

```diff
@@ -240,11 +240,11 @@ def end_session(self,
 
         self._session.video = video
         self._session.end_session(end_state, end_state_reason)
-        token_cost = self._worker.end_session(self._session)
+        token_cost = getattr(self._worker, 'end_session', 'unknown')
```

Comment:
I don't believe this is the correct functionality. We need to call the end_session function here to actually end the session, not retrieve an attribute.

Reply:
Agreed. This was an error in my understanding of the code. I added the code to check if the worker or the worker' session is not present and then logged a message - 

```python
        if self._worker is None or self._worker._session is None:
            return logging.warning("🖇 AgentOps: Cannot end session - no current worker or session")
```

I think the message can be improved to convey more info.